libbeat/cmd/instance/beat.go

```diff
@@ -60,19 +58,11 @@ import (
 	"github.com/elastic/beats/libbeat/metric/system/host"
 	"github.com/elastic/beats/libbeat/monitoring"
 	"github.com/elastic/beats/libbeat/monitoring/report"
+	_ "github.com/elastic/beats/libbeat/monitoring/report/elasticsearch"
 	"github.com/elastic/beats/libbeat/monitoring/report/log"
 	"github.com/elastic/beats/libbeat/outputs/elasticsearch"
 	"github.com/elastic/beats/libbeat/paths"
 	"github.com/elastic/beats/libbeat/plugin"
-	"github.com/elastic/beats/libbeat/publisher/pipeline"
-	svc "github.com/elastic/beats/libbeat/service"
-	"github.com/elastic/beats/libbeat/template"
-	"github.com/elastic/beats/libbeat/version"
-
-	// Register publisher pipeline modules
```

Comment:
This file is importing loads of things, and we are losing these comments for the blank imports. We could maybe move them to an `imports.go` file inside this package.

Reply:
tried it in 071df7728f513ff101effcc14830290fda31f49d, what do you think?

Reply:
Yes, I think I prefer that than having so many blank imports mixed with the rest of imports :+1: 

There could be be yet another option, that is to have multiple import blocks, each one with its own comment so comments are not in some import entries and not in others.

(This new file will need headers).